src/test/java/Homework3/Homework3.java

```diff
@@ -0,0 +1,161 @@
+        By celDropdownLocator = By.xpath(celDropdownXpath);
+        WebElement celDropdownFound = MyDriver.getDriver().findElement(celDropdownLocator);
+        celDropdownFound.click();
+
+        Misc.pause(3);
+
+        // noe get the temp after change it from F to C and verify its changed
+        String tempXpath_2 = "//span[@class='summary swap']";
+        By tempLocator_2 = By.xpath(tempXpath_2);
+        WebElement currentTemp_2 = MyDriver.getDriver().findElement(tempLocator_2);
+        String CurrentTempVal_2 =  currentTemp_2.getText();
+        //Assert.assertEquals(currentTemp_2, currentTempVal,"Temp changed");
+        Assert.assertFalse(currentTempVal.equals(CurrentTempVal_2), "Temp did not change.");
+
+        //MyDriver.quitWindows();
+    }
+
+
+
+    /**
+     * TC-2: facebook.com
+     * Verify current date is selected in the dropdown when user lands on the Create new account form
+     */
+
+    @Test
+    public void verifyCurrentDateIsSelected () {
+
+        // launch facebook page and get the locator for creat new account and click it
+        MyDriver.launchUrlOnNewWindow("https://www.facebook.com/");
+        String createNewAccountXpath = "//a[text()='Create new account']";
+        By createAccountLocator = By.xpath(createNewAccountXpath);
+        WebElement createAccountButton = MyDriver.getDriver().findElement(createAccountLocator);
+        createAccountButton.click();
+
+        Misc.pause(3);
+
+        // get the locator for the current month using select class
+        String currentMonth = ("//select[@id='month']");
+        By currentMonthLocator = By.xpath(currentMonth);
+        WebElement currentMonthFound = MyDriver.getDriver().findElement(currentMonthLocator);
+        Select currMonth = new Select(currentMonthFound);
+        String selectedMonth= currMonth.getFirstSelectedOption().getText();
+
+        Misc.pause(2);
+
+        // get the locator for the current day using select class
+        String currentDay = ("//select[@id='day']");
+        By currentDayLocator = By.xpath(currentDay);
+        WebElement currentDayFound = MyDriver.getDriver().findElement(currentDayLocator);
+        Select currDay = new Select(currentDayFound);
+        String selectedDay= currDay.getFirstSelectedOption().getText();
+
+        Misc.pause(2);
+
+        // get the locator for the current year using select class
+        String currentYear = ("//select[@id='year']");
+        By currentYearLocator = By.xpath(currentYear);
+        WebElement currentYearFound = MyDriver.getDriver().findElement(currentYearLocator);
+        Select currYear = new Select(currentYearFound);
+        String selectedYear= currYear.getFirstSelectedOption().getText();
+        String currentDate = selectedMonth + "/" + selectedDay + "/" + selectedYear;
+
+        Misc.pause(2);
+
+        // get the current date using date class and verify both dates are equals
```

Comment:
Second test case is failing with the message: 

java.lang.AssertionError: Current date is not selected. 
Expected :May/02/2022
Actual   :May/2/2022